Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2_test.py

```diff
@@ -1725,6 +1728,48 @@ def test_test_module(mocker):
     with pytest.raises(Exception) as e:
         module(client)
     assert "Test button cannot be used when using OAuth 2.0" in str(e)
+    
+    
+def test_jwt_checker(mocker):
+    """
+    Given:
+    - private key
+    When:
+    - creating a jwt
+    Then:
+    - (a) that the return type is a string
+    - (b) validate the pem format
+    """
+    import jwt
```

Comment:
General imports can be imported at the beginning of the file with other imports. When we test single command we can import it and just it in the test